burst-kotlin-plugin/src/main/kotlin/app/cash/burst/kotlin/Argument.kt

```diff
@@ -242,6 +245,22 @@ private fun enumValueArguments(
   }
 }
 
+/**
+ * It's possible we'll see two different [IrEnumEntrySymbol] instances for the same enum constant.
+ * This returns a canonical instance by looking up the enum by its name in this [IrPluginContext].
+ *
+ * https://kotlinlang.slack.com/archives/C7L3JB43G/p1732121246270569
```

Comment:
You seem to not be following their advice, though. Seems like we could actually just be using the name to track the default (rather than fully-qualified name) since we know the type will always match.